tensorflow/core/kernels/mkl/mkl_fused_ops_test.cc

```diff
@@ -207,14 +215,26 @@ class MklFusedConv2DOpTest : public OpsTestBase {
   void RunConv2DUnfused(const Tensor& input_data, const Tensor& filter_data,
                         const Tensor& bias_data,
                         const std::vector<string>& fused_ops, Tensor* output,
-                        int stride = 1) {
+                        const int padding, int stride = 1) {
     auto root = tensorflow::Scope::NewRootScope();
     auto input_data_op =
         ops::Const(root.WithOpName("input"), Input::Initializer(input_data));
-    Output next_op = ops::Conv2D(
-        root.WithOpName("conv"), input_data_op,
+    Output next_op = input_data_op;
+
+    if (padding != INVALID_PADDING_VALUE) {
+      Tensor padding_data(DT_INT32, {4, 2});
+      test::FillValues<int32>(&padding_data,
+                              {0, 0, padding, padding, padding, padding, 0, 0});
+      next_op = ops::Pad(root.WithOpName("pad"), next_op,
```

Comment:
Nit: Since this is still about input data, I think the old name makes more sense. (And we can define `next_op` after `if`.)
```suggestion
      input_data_op = ops::Pad(root.WithOpName("pad"), input_data_op,
```